Bottom: 5f1b461d139fa145983d05d2b7ce43f623c76bb5
Top:    ada50cf844fd3dc1a14886bc343dd39e1aaf6cc9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-21 23:44:54 +0000

Refresh of tag-vnodes-with-details-of

---

diff --git a/src/polygon.c b/src/polygon.c
index ddd6e44..1548677 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -516,7 +516,7 @@ frac_circle (PLINE * c, Coord X, Coord Y, Vector v, int fraction)
       e1 = t1;
       v[0] = X + ROUND (e1);
       v[1] = Y + ROUND (e2);
-      poly_InclVertex (c->head.prev, poly_CreateNode (v));
+      poly_InclVertex (c->head.prev, poly_CreateNodeArcApproximation (v, X, Y, radius));
     }
 }
 
@@ -532,6 +532,7 @@ frac_circle2 (PLINE * c, Coord X, Coord Y, Vector v, int fraction)
 {
   double e1, e2, t1;
   int i, range;
+  double radius = sqrt ((v[0] - X) * (v[0] - X) + (v[1] - Y) * (v[1] - Y));
 
   /* move vector to origin */
   e1 = (v[0] - X) * POLY_CIRC_RADIUS_ADJ;
